projects/ngx-wig/src/lib/ngx-wig-component.css

```diff
@@ -253,37 +253,37 @@
         *  prepared via
         *  https://websemantics.uk/tools/svg-to-background-image-conversion/
         *
-        *  .icon.icon-bold
-        *  .icon.icon-check
+        *  .nwicon.nwicon-bold
+        *  .nwicon.nwicon-check
         *  ...
         */
 
-.icon {
+.nwicon {
```

Comment:
Can we say `.nw-icon`?

Reply:
See [my comment](https://github.com/stevermeister/ngx-wig/pull/180#discussion_r989708111) above.